stock_vlm_mgmt/wizards/stock_vlm_task_action.py

```diff
@@ -0,0 +1,186 @@
+# Copyright 2023 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import _, api, fields, models
+
+
+class VlmOperationTaskAction(models.TransientModel):
+    _name = "stock.vlm.task.action"
+    _inherit = ["vlm.tray.cell.position.mixin"]
+    _description = "Actions to perform on vlm task exceptions"
+
+    vlm_task_id = fields.Many2one(
+        string="Selected task", comodel_name="stock.vlm.task", required=True
+    )
+    vlm_task_ids = fields.Many2many(comodel_name="stock.vlm.task")
+    next_vlm_task_id = fields.Many2one(
+        comodel_name="stock.vlm.task", compute="_compute_next_vlm_task_id"
+    )
+    previous_vlm_task_id = fields.Many2one(
+        comodel_name="stock.vlm.task", compute="_compute_next_vlm_task_id"
+    )
+    splitted_vlm_task_id = fields.Many2one(
+        comodel_name="stock.vlm.task", related="vlm_task_id.splitted_vlm_task_id"
+    )
+    image_512 = fields.Image(related="vlm_task_id.product_id.image_512")
+    vlm_tasks_total = fields.Integer(compute="_compute_vlm_tasks_total")
+    vlm_tasks_partial = fields.Integer(compute="_compute_vlm_tasks_total")
+    vlm_tasks_progress = fields.Integer(compute="_compute_vlm_tasks_total")
+    location_id = fields.Many2one(
+        comodel_name="stock.location", related="vlm_task_id.location_id"
+    )
+    tray_id = fields.Many2one(comodel_name="stock.location.vlm.tray")
+    quantity_pending = fields.Float(readonly=True)
+    quantity_done = fields.Float()
+    warning = fields.Char(readonly=True)
+    state = fields.Selection(
+        selection=[
+            ("pending", "Pending"),
+            ("waiting", "Waiting"),
+            ("done", "Done"),
+            ("skipped", "Skipped"),
+            ("edit", "Edit"),
+        ],
+        compute="_compute_state",
+    )
+
+    @api.model
+    def default_get(self, fields):
+        vals = super().default_get(fields)
+        task_id = self.env.context.get("default_vlm_task_id")
+        task = self.env["stock.vlm.task"].browse(task_id)
+        vals.update(
+            {
+                "pos_x": task.pos_x,
+                "pos_y": task.pos_y,
+                "quantity_done": task.quantity_done,
+                "quantity_pending": task.quantity_pending,
+                "tray_id": task.tray_id.id,
+                "location_id": task.location_id.id,
+                "tray_type_id": task.tray_id.tray_type_id.id,
+            }
+        )
+        if not self.env.context.get("default_vlm_task_ids"):
+            vals.update({"vlm_task_ids": [(6, 0, task.ids)]})
```

Comment:
Could you change it to use the [Command](https://github.com/odoo/odoo/blob/0622c210450a69e4331ba38bc1c9b7f389566ff3/odoo/fields.py#L4240
) class? It is more readable.
```suggestion
            vals.update({"vlm_task_ids": [Command.set(task.ids)]})
```

Reply:
Totally agree, done